index.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+<html>
+    <head>
+        <Title>Weather Report</Title>
+        <link rel="stylesheet" href="styles/index.css">
+        
+       
+    </head>
+    <body>
+        <div>
+            <div id="header">
+                <h1>Weather Report</h1>
+                <span>For the lovely city of </span>
+                <h3 id="town-name">My Hometown</h3>
+            </div>
+            <div id="content">
+                <div id="temperature-card">
+                    <h3 id="temperature-title">Temperature</h3>
+                    <br>
+                    <div id="up-arrow" onclick="increaseTemperature()">⬆️</div>
+                    <div id="decreaseTempArrow">⬇️</div>
```

Comment:
These arrows are buttons! It is best practice to use semantic HTML and make these button elements.

In addition, rather than putting the `onclick` event right into the html, we encourage you to use javascript to do this and add the event listener the same way you did for decreaseTemperature

```suggestion
                    <button id="up-arrow" onclick="increaseTemperature()">⬆️</button>
                    <button id="decreaseTempArrow">⬇️</button>
```